scripts/link.sh

```diff
@@ -4,7 +4,7 @@
 #
 
 DEST_DIR="media/econplayground.js/"
-ORIG_DIR="$HOME/public_html/econplayground.js/build/"
+ORIG_DIR="media/js/build/"
```

Comment:
These files in `scripts` can actually be removed now that the repos are merged.

Reply:
Good to know. I'll detete the `/scripts/` directory from the root directory. Should I delete the `/media/econplayground.js/` as well?

Reply:
Yeah

Reply:
Done